shared/actions/chat/inbox.js

```diff
@@ -275,6 +275,19 @@ function* _chatInboxFailedSubSaga(params) {
   })
 
   yield put(Creators.updateInbox(conversation))
+
```

Comment:
don't we only want this to happen if you click into the error, no when we first unbox it?